src/lang/res_ja-JP.rc

```diff
@@ -302,9 +302,9 @@ JAPANEND
     IDS_VERBS + FUNC_MOVE       "ファイル マネージャーは %s を移動できません: %s"
     IDS_VERBS + FUNC_DELETE     "ファイル マネージャーは %s を削除できません: %s"
     IDS_VERBS + FUNC_RENAME     "ファイル マネージャーは %s の名前を変更できません: %s"
-    IDS_VERBS + FUNC_LINK       "ファイルマネージャはシンボリックリンク ％s を作成できません: %s"
-    IDS_VERBS + FUNC_HARD       "ファイルマネージャはハードリンク ％s を作成できません: %s"
-    IDS_VERBS + FUNC_JUNC       "ファイルマネージャはジャンクション ％s を作成できません: %s"
+    IDS_VERBS + FUNC_LINK       "ファイル マネージャーはシンボリックリンク %s を作成できません: %s"

```

Comment:
`ファイル マネージャーは %s のシンボリックリンクを作成できません: %s`? Not sure.

Reply:
> > > Approve. But I wonder why and when these full width `％`'s came into this file...
> > 
> > 
> > This was me, when I introduced symlink creation recently. Google translate gave this to me.
> > Anyway thx for correcting.
> 
> Thank you for your work. Translators are not perfect especially for Asian languages... and do use wrong / disunified 'voices' and unnecessary 'honorific forms' (for a retro program that seldom use it) for Japanese...

It would be great if you could ahead of merge to master also check the few translations in #352.
Are they fine?